### PR TITLE
Change splash image when updater is interrupted while installing.

### PR DESCRIPTION
**Pull request by Ealdwulf**
This PR causes the updater to switch the splash screen during the time when it is installing updates.
It puts it back if it is in any other state.
For future i18n needs, it finds the image based on the locale name.
The image is currently a placeholder.

### CODE REVIEW

File: kano_updater/paths.py [hunk 1]
```diff
@@ -14,3 +14,5 @@
 SYSTEM_VERSION_FILE = '/etc/kanux_version'
 
 STATUS_FILE_PATH = '/var/cache/kano-updater/status.json'
+INTERRUPTED_SPLASH_PATH = '/usr/lib/python2.7/dist-packages/kano_updater/ui/media/images/'
```
Comment (tombettany) [inline kano_updater/paths.py]: Base directory available as [kano_updater.ui.paths.IMAGE_PATH](https://github.com/KanoComputing/kano-updater/blob/master/kano_updater/ui/paths.py#L15).

File: kano_updater/splash.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#
+# splash.py
+#
+# Copyright (C) 2015 Kano Computing Ltd.
```
Comment (tombettany) [inline kano_updater/splash.py]: 2018

File: kano_updater/status.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
 from kano.logging import logger
 
 from kano_updater.paths import STATUS_FILE_PATH
-
+from kano_updater.splash import set_splash_interrupted, clear_splash
```
Comment (tombettany) [inline kano_updater/status.py]: Bump the copyright of this file.

File: kano_updater/paths.py (hunk 1 above)
Comment (tombettany) [reply kano_updater/paths.py]: Also bump copyright.

File: debian/control [hunk 4]
```diff
@@ -25,7 +25,7 @@ Depends:
     python-apt,
     schedtool,
     python-pip,
-    kano-i18n (>= 3.5-1),
+    kano-i18n (>= 3.15-1),
```
Comment (radujipa) [inline debian/control]: *3.15.0

File: kano_updater/paths.py (hunk 1 above)
Comment (Ealdwulf) [reply kano_updater/paths.py]: I wondered why it wasn't there already. Okay, I've eliminated use of this file, so no need to bump copyright.

File: kano_updater/status.py (hunk 3 above)
Comment (Ealdwulf) [reply kano_updater/status.py]: Done.

File: debian/control (hunk 4 above)
Comment (Ealdwulf) [reply debian/control]: Yes - or at least, corresponding to kano-i18n.

**Pull request merged**